src/d3d9/d3d9_device.cpp

```diff
@@ -7309,7 +7314,7 @@ namespace dxvk {
   void D3D9DeviceEx::ResetState(D3DPRESENT_PARAMETERS* pPresentationParameters) {
     SetDepthStencilSurface(nullptr);
 
-    for (uint32_t i = 0; i < caps::MaxSimultaneousRenderTargets; i++)
+    for (uint32_t i = 1; i < caps::MaxSimultaneousRenderTargets; i++)
```

Comment:
This will break Reset.

Reset checks for alive D3DPOOL_DEFAULT textures after calling ResetState, so we cannot hold a reference to RT0 during Reset.